<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettings</class>
 <widget class="QWidget" name="GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="cacheBox">
       <property name="title">
        <string>Cache</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="saveCheckedKeysCheckBox">
            <property name="text">
             <string>Save checked keys in Key ToolBox on exit and restore them on next start.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="clearGpgPasswordCacheCheckBox">
            <property name="text">
             <string>Clear gpg password cache when closing GpgFrontend.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="restoreTextEditorPageCheckBox">
            <property name="text">
             <string>Automatically restore unsaved Text Editor pages after an application crash.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="importConfirmationBox">
       <property name="title">
        <string>Operation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="importConfirmationCheckBox">
            <property name="text">
             <string>Import files dropped on the Key List without confirmation.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="longerKeyExpirationDateCheckBox">
            <property name="text">
             <string>Enable to use longer key expiration date.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="langBox">
       <property name="title">
        <string>Language</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QComboBox" name="langSelectBox">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="langNoteLabel">
            <property name="text">
             <string>NOTE: GpgFrontend will restart automatically if you change the language!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="dataBox">
       <property name="title">
        <string>Data</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="clearAllLogFilesButton">
            <property name="text">
             <string>Clear All Log Files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearAllDataObjectsButton">
            <property name="text">
             <string>Clear All Data Objects Files</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
